### PR TITLE
Allow adding posts and pages with custom titles from the command menu

### PR DESCRIPTION
**Pull request by youknowriad**
Related to https://github.com/WordPress/gutenberg/issues/48457

## What?

This regressed somewhere in the middle of the initial command menu PR but basically the idea is that you can type anything in the command bar and the results proposes results to create pages or posts with what you typed. It was an idea tossed by @jasmussen 

## Testing Instructions

1- Open the site editor
2- Hit cmd+k
3- Type a random post title
4- you click `add post title "your title"` to navigate to the post editor with that custom title prefilled.

### CODE REVIEW

File: packages/edit-site/src/hooks/commands/use-wp-admin-commands.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ const getWPAdminAddCommandLoader = ( postType ) =>
 		const commands = useMemo(
 			() => [
 				{
-					name: 'core/wp-admin/add-' + postType,
+					name: 'core/wp-admin/add-' + postType + '-' + search,
```
Comment (youknowriad) [inline packages/edit-site/src/hooks/commands/use-wp-admin-commands.js:47]: The command menu package we use, rely on the "value" prop (which is the name of the command for us) to actually filter the results. This command was returned by this hook but was hidden by the search algorithm.

**Review by jasmussen — APPROVED** [review]
Since this was mostly a regression, let's land this. Works for me:

Screenshot 2023-04-18 at 15 24 50


Screenshot 2023-04-18 at 15 21 50

We should probably come back to this, as we've discussed a "new page" modal that allows you to draft pages quickly, that we can unify on (https://github.com/WordPress/gutenberg/issues/49873#issuecomment-1512565294) — and it would be nice if this commandbar hooked into that as well. Both in terms of connecting with it (type "Contact", press Add new page, and it seamlessly transitioned into the new page modal with the title field pre-filled), but also design/position wise, so it can kind of "feel" like a single modal.

**Comment by oandregal** [comment]
@youknowriad I'm not sure how to open the command menu. Using ctrl+k sets the focus in the browser address bar. Other keyboard combinations do nothing. My environment: linux (ubuntu) + chrome or firefox browser.

**Review by ntsekouras — APPROVED** [review]

**Comment by youknowriad** [comment]
@oandregal Since we're using the keyboard shortcuts package, the focus needs to be within the page when you hit the ctrl + k. Can you try that?

**Comment by oandregal** [comment]
@youknowriad Apparently, CTRL+K is a browser shortcut ([chrome](https://support.google.com/chrome/answer/157179#zippy=%2Caddress-bar-shortcuts), [firefox](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly#w_search)) in Windows and Linux.

[Gravação de ecrã a partir de 19-04-2023 10:05:46.webm](https://user-images.githubusercontent.com/583546/233010205-752cd79b-31cd-4991-b2ed-1de4fb8ca468.webm)

I've also noticed that CTRL+K is also used for opening the link panel, when the focus is on the editor canvas:

[Gravação de ecrã a partir de 19-04-2023 10:12:29.webm](https://user-images.githubusercontent.com/583546/233011880-d3dbd459-cad0-46ed-910e-9d6b9a21301d.webm)

**Comment by youknowriad** [comment]
@oandregal yeah, we're aware of this but at least in Mac OS Firefox, I managed to prevent the default browser behavior.  For the link panel, one of the ideas was to actually use a contextual command menu to insert the link there. (unify both) 

Do you think you can take a look to see if you can find a way to prevent the default browser behavior in Linux? also cmd+k is a common pattern for this kind of menus across the web. In fact the lib is called cmdk https://github.com/pacocoursey/cmdk 

That said, this discussion is independent from the current PR, so I'm going to merge this one and I think we should continue exploring a solution to prevent the default browser behavior.

**Pull request merged**